spec/models/leitner_method_spec.rb

```diff
@@ -47,84 +47,35 @@
-    end
-
-    context "field wrong_counter is equal 2 and correct_counter is equal 1" do
-      let(:card) { FactoryGirl.create :card, correct_counter: 1, wrong_counter: 1, review_date: DateTime.now }
-      it { expect(card.correct_counter).to eq 1 }
-      it { expect(card.wrong_counter).to eq 2 }
-      it { expect(card.review_date).to eq DateTime.now }
-    end
-
-    context "field wrong_counter is equal 1 and correct_counter is equal 2" do
-      let(:card) { FactoryGirl.create :card, correct_counter: 2, wrong_counter: 0, review_date: DateTime.now }
-      it { expect(card.correct_counter).to eq 2 }
-      it { expect(card.wrong_counter).to eq 1 }
-      it { expect(card.review_date).to eq DateTime.now }
-    end
-
-    context "field wrong_counter is equal 2 and correct_counter is equal 2" do
-      let(:card) { FactoryGirl.create :card, correct_counter: 2, wrong_counter: 1, review_date: DateTime.now }
-      it { expect(card.correct_counter).to eq 2 }
-      it { expect(card.wrong_counter).to eq 2 }
-      it { expect(card.review_date).to eq DateTime.now }
+      card.verify_translation("qwerty")
     end
 
-    context "field wrong_counter is equal 1 and correct_counter is equal 3" do
-      let(:card) { FactoryGirl.create :card, correct_counter: 3, wrong_counter: 0, review_date: DateTime.now }
-      it { expect(card.correct_counter).to eq 3 }
-      it { expect(card.wrong_counter).to eq 1 }
-      it { expect(card.review_date).to eq DateTime.now }
-    end
-
-    context "field wrong_counter is equal 2 and correct_counter is equal 3" do
-      let(:card) { FactoryGirl.create :card, correct_counter: 3, wrong_counter: 1, review_date: DateTime.now }
-      it { expect(card.correct_counter).to eq 3 }
-      it { expect(card.wrong_counter).to eq 2 }
-      it { expect(card.review_date).to eq DateTime.now }
-    end
-
-    context "field wrong_counter is equal 1 and correct_counter is equal 4" do
-      let(:card) { FactoryGirl.create :card, correct_counter: 4, wrong_counter: 0, review_date: DateTime.now }
-      it { expect(card.correct_counter).to eq 4 }
-      it { expect(card.wrong_counter).to eq 1 }
-      it { expect(card.review_date).to eq DateTime.now }
-    end
-
-    context "field wrong_counter is equal 2 and correct_counter is equal 4" do
-      let(:card) { FactoryGirl.create :card, correct_counter: 4, wrong_counter: 1, review_date: DateTime.now }
-      it { expect(card.correct_counter).to eq 4 }
-      it { expect(card.wrong_counter).to eq 2 }
-      it { expect(card.review_date).to eq DateTime.now }
-    end
-
-    context "field wrong_counter is equal 1 and correct_counter is equal 5" do
-      let(:card) { FactoryGirl.create :card, correct_counter: 5, wrong_counter: 0, review_date: DateTime.now }
-      it { expect(card.correct_counter).to eq 5 }
+    context "field wrong_counter is equal 1" do
+      let(:card) { FactoryGirl.create :card, wrong_counter: 0, review_date: DateTime.now }
```

Comment:
Line is too long. [90/80]<br>Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.